pkg/agent/openflow/pipeline.go

```diff
@@ -915,13 +915,8 @@ func (c *client) serviceLBBypassFlows(ipProtocol binding.Protocol) []binding.Flo
 }
 
 // l2ForwardCalcFlow generates the flow that matches dst MAC and loads ofPort to reg.
-func (c *client) l2ForwardCalcFlow(dstMAC net.HardwareAddr, ofPort uint32, skipIngressRules bool, category cookie.Category) binding.Flow {
+func (c *client) l2ForwardCalcFlow(dstMAC net.HardwareAddr, ofPort uint32, nextTable binding.TableIDType, category cookie.Category) binding.Flow {
```

Comment:
I think this is not simpler than the original method that exposes a bool to determine whether ingress should be skipped. And InstallGatewayFlows should use IngressMetricTable as next table too, which should be the root cause why noencap test failed.
Would it fix it if we just use IngressMetricTable as the default next table of l2ForwardingCalcTable?